### ISSUE TITLE
`README` in markdown instead of `.rst`?

### ISSUE THREAD

**Issue by darribas** [pysal/pointpats#24]
While working on the [Notebooks book](https://github.com/pysal/notebooks/labels/nbs2.0) project, I've realised that the README file for this project is structured in `.rst`. Is there a reason for this? If not, would it be possible to convert it markdown?  I'm currently converting with `pandoc` but would probably be better to be consistent across de federation?

**Comment by weikang9009** [comment]
I do not think there is a specific reason for only having the README file in `.rst`. But if `.md` is preferred in the notebooks book project and the meta package building, it makes sense to convert `README.rst` to `.md`. I also think it might make sense to require `README.md` in pysal submodules and list it in the [submodule-contract](http://pysal.org/getting_started.html#submodule-contract) since other packages such as [spreg](https://github.com/pysal/spreg) has README.rst only as well. Would it also be reasonable to require `README.md` only in submodules as `README.rst` seems to be just a duplicate in most pysal packages which have both files (e.g. [spaghetti](https://github.com/pysal/spaghetti), [giddy](https://github.com/pysal/giddy))?

**Comment by sjsrey** [comment]
If my memory is correct, I think there are issues with the listing on pypi for packages, and my guess is that is why we had rst and md in some cases?

**Comment by weikang9009** [comment]
Based on the latest [Python Packaging User Guide](https://packaging.python.org/guides/making-a-pypi-friendly-readme/), it seems that we can set the parameter`long_description_content_type` to be `'text/markdown'` in `setup()` to use the content of `README.md` for the package long_description:

```
from setuptools import setup

# read the contents of your README file
from os import path
this_directory = path.abspath(path.dirname(__file__))
with open(path.join(this_directory, 'README.md'), encoding='utf-8') as f:
    long_description = f.read()

setup(
    name='an_example_package',
    # other arguments omitted
    long_description=long_description,
    long_description_content_type='text/markdown'
)
```

**Comment by darribas** [comment]
Closed as addressed in #26
